docs/ota-client-guide/modules/ROOT/pages/release-process.adoc

```diff
@@ -82,9 +82,14 @@ Once the release is ready on github, it should be edited to include a link to th
 
 The https://github.com/advancedtelematic/homebrew-otaconnect/blob/master/aktualizr.rb[homebrew aktualizr recipe] should be updated with the new release.
 
-There is a github CI job that will automatically update the recipe, build a bottle, and upload the bottle to the release page when a new release is created. The one thing it doesn't do is make the pull request to update the recipe in the tap.
-
-You'll find the new recipe attached to the CI run for the release--it should be the latest run on the https://github.com/advancedtelematic/aktualizr/actions[Actions tab]. You can download the file from there, and submit the changes as a pull request to the https://github.com/advancedtelematic/homebrew-otaconnect/[homebrew-otaconnect repo].
+There is the gitlab CI job that will automatically update the recipe, build a bottle,
+and upload the bottle to the github release page when a new release is created.
+The one thing it doesn't do is make the pull request to update the recipe in the tap, get the pull request approval
+and merge it into the master branch.
+
+You will find the new recipe attached to the gitlab CI job named `build-osx-release` that should be triggered in
+the scope of the latest https://main.gitlab.in.here.com/olp/edge/ota/connect/client/aktualizr/pipelines?scope=tags&page=1[release CI pipeline].
+You can download the file from there, and submit the changes as a pull request to the https://github.com/advancedtelematic/homebrew-otaconnect/[homebrew-otaconnect repo].
```

Comment:
BTW, we generally try to avoid these unnecessary line breaks (they aren't rendered), but I'm not going to hold up the PR for that.

Reply:
It's just inconvenient to read it in an editor if a line is really long.

Reply:
Depends on the text editor, and the line breaks can cause some problems with rendering and formatting. Our tech writer team and historically had strong feelings about this that I happen to share.

Reply:
There's no modern text editor that doesn't allow you to configure wrapping in whatever way works best for you, whereas putting in hard linebreaks means imposing one particular column width on everyone else. Also makes diffs for textual content worse.

Reply:
OK, I'll revert it to the one-line version, my "modern" editor is tuned for code writing, not for documentation :)